maxim/apis/maxim_apis.py

```
@@ -874,9 +876,9 @@ def _process_file_attachment(self, attachment: Union[FileDataAttachment, FileAtt
                 mime_type = attachment.mime_type or 'application/octet-stream'
                 with open(attachment.path, 'rb') as f:
                     file_data = f.read()
-            
+
             return file_data, mime_type, size
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)